src/components/Header.jsx

```diff
@@ -29,4 +30,8 @@ class Header extends Component {
   }
 }
 
+Header.propTypes = {
+  triggerHomePageMenu: PropTypes.func
```

Comment:
.isRequired  (stopping with individual ones now)

Reply:
Should be addressed in follow-up ticket #211